### ISSUE TITLE
Screen review shows inaccurate information in some cases

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#6221]
I've seen several cases where the screen review mode shows inaccurate results. Some examples:
## Notepad

Str:
1. Open notepad
2. Maximize the screen
3. Use screen review to read the Window

Expected: The menu bar is shown only once
Actual: The menu bar is shown twice. fg.displayText contains menu bar once, fg.parent.displayText contains it twice
## Test application

I created a test application in Delphi7, [download here](http://www.babbage.com/NVDA/testapp.exe)

Str:
1. Start application
2. Maximize the window
3. Clear the multiline edit box
4. Read the window with screen review

Expected: Screen review only reveals NeverEndingStory
Actual: The previous contents of the edit area are still being revealed.

I wonder, is this something related to Windows, or might there be a bug in NVDA's displayModel module?

**Comment by leonardder** [comment]
Closing in favour of #5727, even though this ticket might have some valuable information for that ticket as well.
